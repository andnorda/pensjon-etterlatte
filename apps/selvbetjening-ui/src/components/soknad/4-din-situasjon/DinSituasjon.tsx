import SoknadSteg from "../../../typer/SoknadSteg";
import { RadioProps, SkjemaGruppe } from "nav-frontend-skjema";
import { ISituasjon, JobbStatus } from "../../../typer/situasjon";
import { FormProvider, useForm } from "react-hook-form";
import { ActionTypes } from "../../../context/soknad/soknad";
import { useSoknadContext } from "../../../context/soknad/SoknadContext";
import NavaerendeArbeidsforhold from "./fragmenter/NavaerendeArbeidsforhold";
import Feilmeldinger from "../../felles/Feilmeldinger";
import AndreYtelser from "./fragmenter/AndreYtelser";
import HoeyesteUtdanning from "./fragmenter/HoeyesteUtdanning";
import Navigasjon from "../../felles/Navigasjon";
import { useTranslation } from "react-i18next";
import UnderUtdanning from "./fragmenter/UnderUtdanning";
import { RHFInput } from "../../felles/RHFInput";
import HvorforSpoerVi from "../../felles/HvorforSpoerVi";
import useEffectOnce from "../../../hooks/useEffectOnce";
import { isEmpty } from "lodash";
import { BodyLong, Title } from "@navikt/ds-react";
import { RHFCheckboksPanelGruppe } from "../../felles/RHFCheckboksPanelGruppe";

const DinSituasjon: SoknadSteg = ({ neste, forrige }) => {
    const { t } = useTranslation();

    const { state, dispatch } = useSoknadContext();

    const methods = useForm<ISituasjon>({
        defaultValues: state.dinSituasjon || {},
        shouldUnregister: true,
    });

    useEffectOnce(() => {
        methods.reset(state.dinSituasjon);
    }, !isEmpty(state.dinSituasjon));

    const {
        handleSubmit,
        formState: { errors },
        watch,
    } = methods;

    const lagre = (data: ISituasjon) => {
        dispatch({ type: ActionTypes.OPPDATER_DIN_SITUASJON, payload: data });
        neste!!();
    };

    const jobbStatus = watch("jobbStatus");
    return (
        <FormProvider {...methods}>
            <form>
                <SkjemaGruppe>
                    <Title size={"m"} className={"center"}>
                        {t("dinSituasjon.tittel")}
                    </Title>
                </SkjemaGruppe>

                <SkjemaGruppe>
                    <Title size={"s"}>{t("dinSituasjon.undertittel")}</Title>
                    <BodyLong>{t("dinSituasjon.ingress")}</BodyLong>
                </SkjemaGruppe>

                <RHFCheckboksPanelGruppe
                    name={"jobbStatus"}
                    legend={t("dinSituasjon.jobbStatus")}
                    checkboxes={Object.values(JobbStatus).map((value) => {
                        return { label: t(value), value } as RadioProps;
                    })}
                />

                {(jobbStatus?.includes(JobbStatus.selvstendig) || jobbStatus?.includes(JobbStatus.arbeidstaker)) && (
                    <NavaerendeArbeidsforhold />
                )}

                {jobbStatus?.includes(JobbStatus.underUtdanning) && <UnderUtdanning />}

                {jobbStatus?.includes(JobbStatus.ingen) && (
                    <SkjemaGruppe>
                        <RHFInput name={"ingenJobbBeskrivelse"} label={t("dinSituasjon.ingenJobbBeskrivelse")} />
                    </SkjemaGruppe>
                )}

                {/* <TidligereArbeidsforhold />*/}

                <HoeyesteUtdanning />

                <AndreYtelser />

                <Feilmeldinger errors={errors} />

                <Navigasjon forrige={{ onClick: forrige }} neste={{ onClick: handleSubmit(lagre) }} />
            </form>
        </FormProvider>
    );
};

export default DinSituasjon;
