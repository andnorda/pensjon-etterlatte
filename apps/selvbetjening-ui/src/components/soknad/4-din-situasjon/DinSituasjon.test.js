import { render } from "@testing-library/react";
import * as JSutils from "nav-frontend-js-utils";
import DinSituasjon from "./DinSituasjon";

jest.mock("react-i18next", () => ({
    // this mock makes sure any components using the translate hook can use it without a warning being shown
    useTranslation: () => {
        return {
            t: (str) => str,
            i18n: {
                changeLanguage: () => new Promise(() => {}),
            },
        };
    },
}));

const defaultValues = {
    jobbStatus: ["jobbStatus.arbeidstaker"],
    utdanning: {
        hoyesteFullfoerteUtdanning: "utdanning.mastergrad",
    },
    andreYtelser: {
        kravOmAnnenStonad: {
            svar: "Ja",
            beskrivelse: "Barnepensjon",
        },
        annenPensjon: {
            svar: "Ja",
            beskrivelse: "Skandia",
        },
        mottarPensjonUtland: {
            svar: "Ja",
            hvaSlagsPensjon: "Polsk Uførepensjon",
            fraHvilketLand: "Polen",
            bruttobeloepPrAar: "4000 PLN",
        },
    },
    arbeidsforhold: [
        {
            arbeidsgiver: "Potetskreller AS",
            ansettelsesforhold: "stillingType.midlertidig",
            stillingsprosent: "100%",
            forventerEndretInntekt: {
                svar: "Ja",
                beskrivelse: "150 000",
            },
        },
    ],
};

jest.mock("../../../context/soknad/SoknadContext", () => ({
    useSoknadContext: () => ({
        state: { dinSituasjon: defaultValues },
        dispatch: jest.fn(),
    }),
}));

JSutils.guid = jest.fn(() => "123");
describe("Om den avdøde", () => {
    it("Snapshot", () => {
        const { container } = render(<DinSituasjon />);
        expect(container).toMatchSnapshot();
    });
});
