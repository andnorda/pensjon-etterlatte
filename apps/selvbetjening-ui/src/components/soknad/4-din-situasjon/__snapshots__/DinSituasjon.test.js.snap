// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Om den avdøde Snapshot 1`] = `
<div>
  <form>
    <fieldset
      aria-invalid="false"
      class="skjemagruppe"
    >
      <h1
        class="center navds-title navds-title--m"
      >
        dinSituasjon.tittel
      </h1>
      <div
        aria-live="polite"
      />
    </fieldset>
    <fieldset
      aria-invalid="false"
      class="skjemagruppe"
    >
      <h1
        class="navds-title navds-title--s"
      >
        dinSituasjon.undertittel
      </h1>
      <p
        class="navds-body-long"
      >
        dinSituasjon.ingress
      </p>
      <div
        aria-live="polite"
      />
    </fieldset>
    <fieldset
      aria-invalid="false"
      class="skjemagruppe checkboxgruppe inputPanelGruppe"
    >
      <legend
        class="skjemagruppe__legend"
      >
        dinSituasjon.jobbStatus
      </legend>
      <label
        class="inputPanel checkboksPanel inputPanel--checked"
      >
        <input
          aria-checked="true"
          aria-invalid="false"
          checked=""
          class="inputPanel__field"
          type="checkbox"
          value="jobbStatus.arbeidstaker"
        />
        <span
          class="inputPanel__label"
        >
          jobbStatus.arbeidstaker
        </span>
      </label>
      <label
        class="inputPanel checkboksPanel"
      >
        <input
          aria-checked="false"
          aria-invalid="false"
          class="inputPanel__field"
          type="checkbox"
          value="jobbStatus.selvstendig"
        />
        <span
          class="inputPanel__label"
        >
          jobbStatus.selvstendig
        </span>
      </label>
      <label
        class="inputPanel checkboksPanel"
      >
        <input
          aria-checked="false"
          aria-invalid="false"
          class="inputPanel__field"
          type="checkbox"
          value="jobbStatus.underUtdanning"
        />
        <span
          class="inputPanel__label"
        >
          jobbStatus.underUtdanning
        </span>
      </label>
      <label
        class="inputPanel checkboksPanel"
      >
        <input
          aria-checked="false"
          aria-invalid="false"
          class="inputPanel__field"
          type="checkbox"
          value="jobbStatus.ingen"
        />
        <span
          class="inputPanel__label"
        >
          jobbStatus.ingen
        </span>
      </label>
      <div
        aria-live="polite"
      />
    </fieldset>
    <fieldset
      aria-invalid="false"
      class="skjemagruppe skjemagruppering"
    >
      <fieldset
        aria-invalid="false"
        class="skjemagruppe"
      >
        <h1
          class="navds-title navds-title--s"
        >
          dinSituasjon.arbeidsforhold.tittel
        </h1>
        <div
          aria-live="polite"
        />
      </fieldset>
      <div
        class="navds-panel luft-under navds-panel--border"
      >
        <div
          class="kol-75"
        >
          <div
            class="skjemaelement"
          >
            <label
              class="skjemaelement__label"
              for="arbeidsforhold[0].arbeidsgiver"
            >
              dinSituasjon.arbeidsforhold.arbeidsgiver
            </label>
            <input
              aria-invalid="false"
              class="skjemaelement__input input--fullbredde"
              id="arbeidsforhold[0].arbeidsgiver"
              type="text"
              value="Potetskreller AS"
            />
          </div>
        </div>
        <fieldset
          aria-invalid="false"
          class="skjemagruppe rad"
        >
          <div
            id="arbeidsforhold[0].ansettelsesforhold"
          >
            <div
              class="skjemaelement"
            >
              <label
                class="skjemaelement__label"
              >
                dinSituasjon.arbeidsforhold.ansettelsesforhold
              </label>
              <div
                class="selectContainer input--l"
              >
                <select
                  aria-invalid="false"
                  class="skjemaelement__input"
                >
                  <option>
                    Velg ...
                  </option>
                  <option
                    value="stillingType.fast"
                  >
                    stillingType.fast
                  </option>
                  <option
                    value="stillingType.midlertidig"
                  >
                    stillingType.midlertidig
                  </option>
                  <option
                    value="stillingType.sesongarbeid"
                  >
                    stillingType.sesongarbeid
                  </option>
                </select>
              </div>
            </div>
          </div>
          <div
            class="skjemaelement"
          >
            <label
              class="skjemaelement__label"
              for="arbeidsforhold[0].stillingsprosent"
            >
              dinSituasjon.arbeidsforhold.stillingsprosent
            </label>
            <input
              aria-invalid="false"
              class="skjemaelement__input input--fullbredde"
              id="arbeidsforhold[0].stillingsprosent"
              placeholder="dinSituasjon.arbeidsforhold.stillingsprosentPlaceholder"
              type="text"
              value="100%"
            />
          </div>
          <div
            aria-live="polite"
          />
        </fieldset>
        <div
          id="arbeidsforhold[0].forventerEndretInntekt.svar"
        >
          <fieldset
            aria-invalid="false"
            class="skjemagruppe inputPanelGruppe inline"
          >
            <legend
              class="skjemagruppe__legend"
            >
              dinSituasjon.arbeidsforhold.forventerEndretInntekt.svar
            </legend>
            <div
              class="inputPanelGruppe__inner"
            >
              <label
                class="inputPanel radioPanel inputPanel--checked"
              >
                <input
                  aria-checked="true"
                  aria-invalid="false"
                  checked=""
                  class="inputPanel__field"
                  name="arbeidsforhold[0].forventerEndretInntekt.svar"
                  type="radio"
                  value="Ja"
                />
                <span
                  class="inputPanel__label"
                >
                  Ja
                </span>
              </label>
              <label
                class="inputPanel radioPanel"
              >
                <input
                  aria-checked="false"
                  aria-invalid="false"
                  class="inputPanel__field"
                  name="arbeidsforhold[0].forventerEndretInntekt.svar"
                  type="radio"
                  value="Nei"
                />
                <span
                  class="inputPanel__label"
                >
                  Nei
                </span>
              </label>
              <label
                class="inputPanel radioPanel"
              >
                <input
                  aria-checked="false"
                  aria-invalid="false"
                  class="inputPanel__field"
                  name="arbeidsforhold[0].forventerEndretInntekt.svar"
                  type="radio"
                  value="Vet ikke"
                />
                <span
                  class="inputPanel__label"
                >
                  Vet ikke
                </span>
              </label>
            </div>
            <div
              aria-live="polite"
            />
          </fieldset>
        </div>
        <div
          class="skjemaelement"
        >
          <label
            class="skjemaelement__label"
            for="arbeidsforhold[0].forventerEndretInntekt.beskrivelse"
          >
            dinSituasjon.arbeidsforhold.forventerEndretInntekt.beskrivelse
          </label>
          <input
            aria-invalid="false"
            class="skjemaelement__input input--s"
            id="arbeidsforhold[0].forventerEndretInntekt.beskrivelse"
            type="text"
            value="150 000"
          />
        </div>
      </div>
      <button
        class="navds-button navds-button--secondary navds-button--m navds-body-short"
        type="button"
      >
        + 
        knapp.leggTil
      </button>
      <div
        aria-live="polite"
      />
    </fieldset>
    <fieldset
      aria-invalid="false"
      class="skjemagruppe skjemagruppering"
    >
      <div
        id="utdanning.hoyesteFullfoerteUtdanning"
      >
        <fieldset
          aria-invalid="false"
          class="skjemagruppe inputPanelGruppe"
        >
          <legend
            class="skjemagruppe__legend"
          >
            dinSituasjon.utdanning.hoyesteFullfoerteUtdanning
          </legend>
          <div
            class="inputPanelGruppe__inner"
          >
            <label
              class="inputPanel radioPanel"
            >
              <input
                aria-checked="false"
                aria-invalid="false"
                class="inputPanel__field"
                name="utdanning.hoyesteFullfoerteUtdanning"
                type="radio"
                value="utdanning.grunnskole"
              />
              <span
                class="inputPanel__label"
              >
                utdanning.grunnskole
              </span>
            </label>
            <label
              class="inputPanel radioPanel"
            >
              <input
                aria-checked="false"
                aria-invalid="false"
                class="inputPanel__field"
                name="utdanning.hoyesteFullfoerteUtdanning"
                type="radio"
                value="utdanning.videregående"
              />
              <span
                class="inputPanel__label"
              >
                utdanning.videregående
              </span>
            </label>
            <label
              class="inputPanel radioPanel"
            >
              <input
                aria-checked="false"
                aria-invalid="false"
                class="inputPanel__field"
                name="utdanning.hoyesteFullfoerteUtdanning"
                type="radio"
                value="utdanning.bachelorgrad"
              />
              <span
                class="inputPanel__label"
              >
                utdanning.bachelorgrad
              </span>
            </label>
            <label
              class="inputPanel radioPanel inputPanel--checked"
            >
              <input
                aria-checked="true"
                aria-invalid="false"
                checked=""
                class="inputPanel__field"
                name="utdanning.hoyesteFullfoerteUtdanning"
                type="radio"
                value="utdanning.mastergrad"
              />
              <span
                class="inputPanel__label"
              >
                utdanning.mastergrad
              </span>
            </label>
            <label
              class="inputPanel radioPanel"
            >
              <input
                aria-checked="false"
                aria-invalid="false"
                class="inputPanel__field"
                name="utdanning.hoyesteFullfoerteUtdanning"
                type="radio"
                value="utdanning.ingen"
              />
              <span
                class="inputPanel__label"
              >
                utdanning.ingen
              </span>
            </label>
            <label
              class="inputPanel radioPanel"
            >
              <input
                aria-checked="false"
                aria-invalid="false"
                class="inputPanel__field"
                name="utdanning.hoyesteFullfoerteUtdanning"
                type="radio"
                value="utdanning.annen"
              />
              <span
                class="inputPanel__label"
              >
                utdanning.annen
              </span>
            </label>
          </div>
          <div
            aria-live="polite"
          />
        </fieldset>
      </div>
      <div
        aria-live="polite"
      />
    </fieldset>
    <fieldset
      aria-invalid="false"
      class="skjemagruppe skjemagruppering"
    >
      <fieldset
        aria-invalid="false"
        class="skjemagruppe"
      >
        <h1
          class="navds-title navds-title--s"
        >
          dinSituasjon.andreYtelser.tittel
        </h1>
        <p
          class="navds-body-long"
        >
          dinSituasjon.andreYtelser.ingress
        </p>
        <div
          aria-live="polite"
        />
      </fieldset>
      <div
        id="andreYtelser.kravOmAnnenStonad.svar"
      >
        <fieldset
          aria-invalid="false"
          class="skjemagruppe inputPanelGruppe inline"
        >
          <legend
            class="skjemagruppe__legend"
          >
            dinSituasjon.andreYtelser.kravOmAnnenStonad.svar
          </legend>
          <div
            class="inputPanelGruppe__inner"
          >
            <label
              class="inputPanel radioPanel inputPanel--checked"
            >
              <input
                aria-checked="true"
                aria-invalid="false"
                checked=""
                class="inputPanel__field"
                name="andreYtelser.kravOmAnnenStonad.svar"
                type="radio"
                value="Ja"
              />
              <span
                class="inputPanel__label"
              >
                Ja
              </span>
            </label>
            <label
              class="inputPanel radioPanel"
            >
              <input
                aria-checked="false"
                aria-invalid="false"
                class="inputPanel__field"
                name="andreYtelser.kravOmAnnenStonad.svar"
                type="radio"
                value="Nei"
              />
              <span
                class="inputPanel__label"
              >
                Nei
              </span>
            </label>
          </div>
          <div
            aria-live="polite"
          />
        </fieldset>
      </div>
      <div>
        <div
          class="skjemaelement"
        >
          <label
            class="skjemaelement__label"
            for="andreYtelser.kravOmAnnenStonad.beskrivelse"
          >
            dinSituasjon.andreYtelser.kravOmAnnenStonad.beskrivelse
          </label>
          <input
            aria-invalid="false"
            class="skjemaelement__input input--xxl"
            id="andreYtelser.kravOmAnnenStonad.beskrivelse"
            placeholder="dinSituasjon.andreYtelser.kravOmAnnenStonad.placeholder"
            type="text"
            value="Barnepensjon"
          />
        </div>
      </div>
      <div
        aria-live="polite"
      />
    </fieldset>
    <fieldset
      aria-invalid="false"
      class="skjemagruppe skjemagruppering"
    >
      <div
        id="andreYtelser.annenPensjon.svar"
      >
        <fieldset
          aria-invalid="false"
          class="skjemagruppe inputPanelGruppe inline"
        >
          <legend
            class="skjemagruppe__legend"
          >
            dinSituasjon.andreYtelser.annenPensjon.svar
          </legend>
          <div
            class="inputPanelGruppe__inner"
          >
            <label
              class="inputPanel radioPanel inputPanel--checked"
            >
              <input
                aria-checked="true"
                aria-invalid="false"
                checked=""
                class="inputPanel__field"
                name="andreYtelser.annenPensjon.svar"
                type="radio"
                value="Ja"
              />
              <span
                class="inputPanel__label"
              >
                Ja
              </span>
            </label>
            <label
              class="inputPanel radioPanel"
            >
              <input
                aria-checked="false"
                aria-invalid="false"
                class="inputPanel__field"
                name="andreYtelser.annenPensjon.svar"
                type="radio"
                value="Nei"
              />
              <span
                class="inputPanel__label"
              >
                Nei
              </span>
            </label>
          </div>
          <div
            aria-live="polite"
          />
        </fieldset>
      </div>
      <div>
        <div
          class="skjemaelement"
        >
          <label
            class="skjemaelement__label"
            for="andreYtelser.annenPensjon.beskrivelse"
          >
            dinSituasjon.andreYtelser.annenPensjon.beskrivelse
          </label>
          <input
            aria-invalid="false"
            class="skjemaelement__input input--xxl"
            id="andreYtelser.annenPensjon.beskrivelse"
            placeholder="dinSituasjon.andreYtelser.annenPensjon.placeholder"
            type="text"
            value="Skandia"
          />
        </div>
      </div>
      <div
        aria-live="polite"
      />
    </fieldset>
    <fieldset
      aria-invalid="false"
      class="skjemagruppe skjemagruppering"
    >
      <div
        id="andreYtelser.mottarPensjonUtland.svar"
      >
        <fieldset
          aria-invalid="false"
          class="skjemagruppe inputPanelGruppe inline"
        >
          <legend
            class="skjemagruppe__legend"
          >
            dinSituasjon.andreYtelser.mottarPensjonUtland.svar
          </legend>
          <div
            class="skjemagruppe__description"
          >
            <div
              class="hvorforPanel"
            >
              <button
                class="hvorforPanel__toggle"
                data-testid="hvorforPanel_toggle"
                type="button"
              >
                <span>
                  hvorforSpoerVi
                </span>
                <span>
                  <svg
                    fill="none"
                    height="1em"
                    viewBox="0 0 24 24"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      clip-rule="evenodd"
                      d="M12 17L3 8.429 4.5 7l7.5 7.143L19.5 7 21 8.429 12 17z"
                      fill="currentColor"
                      fill-rule="evenodd"
                    />
                  </svg>
                </span>
              </button>
            </div>
          </div>
          <div
            class="inputPanelGruppe__inner"
          >
            <label
              class="inputPanel radioPanel inputPanel--checked"
            >
              <input
                aria-checked="true"
                aria-invalid="false"
                checked=""
                class="inputPanel__field"
                name="andreYtelser.mottarPensjonUtland.svar"
                type="radio"
                value="Ja"
              />
              <span
                class="inputPanel__label"
              >
                Ja
              </span>
            </label>
            <label
              class="inputPanel radioPanel"
            >
              <input
                aria-checked="false"
                aria-invalid="false"
                class="inputPanel__field"
                name="andreYtelser.mottarPensjonUtland.svar"
                type="radio"
                value="Nei"
              />
              <span
                class="inputPanel__label"
              >
                Nei
              </span>
            </label>
          </div>
          <div
            aria-live="polite"
          />
        </fieldset>
      </div>
      <div>
        <div
          class="skjemaelement"
        >
          <label
            class="skjemaelement__label"
            for="andreYtelser.mottarPensjonUtland.hvaSlagsPensjon"
          >
            dinSituasjon.andreYtelser.mottarPensjonUtland.hvaSlagsPensjon
          </label>
          <input
            aria-invalid="false"
            class="skjemaelement__input input--xxl"
            id="andreYtelser.mottarPensjonUtland.hvaSlagsPensjon"
            placeholder="dinSituasjon.andreYtelser.mottarPensjonUtland.hvaSlagsPensjonPlaceholder"
            type="text"
            value="Polsk Uførepensjon"
          />
        </div>
      </div>
      <div>
        <div
          class="skjemaelement"
        >
          <label
            class="skjemaelement__label"
            for="andreYtelser.mottarPensjonUtland.fraHvilketLand"
          >
            dinSituasjon.andreYtelser.mottarPensjonUtland.fraHvilketLand
          </label>
          <input
            aria-invalid="false"
            class="skjemaelement__input input--l"
            id="andreYtelser.mottarPensjonUtland.fraHvilketLand"
            type="text"
            value="Polen"
          />
        </div>
      </div>
      <div>
        <div
          class="skjemaelement"
        >
          <label
            class="skjemaelement__label"
            for="andreYtelser.mottarPensjonUtland.bruttobeloepPrAar"
          >
            dinSituasjon.andreYtelser.mottarPensjonUtland.bruttobeloepPrAar
          </label>
          <input
            aria-invalid="false"
            class="skjemaelement__input input--l"
            id="andreYtelser.mottarPensjonUtland.bruttobeloepPrAar"
            type="text"
            value="4000 PLN"
          />
        </div>
      </div>
      <div
        aria-live="polite"
      />
    </fieldset>
    <fieldset
      aria-invalid="false"
      class="skjemagruppe navigasjon-footer"
    >
      <div
        class="navigasjon-rad"
      >
        <button
          class="navds-button navds-button--primary navds-button--m navds-body-short"
          type="button"
        >
          knapp.tilbake
        </button>
        <button
          class="navds-button navds-button--action navds-button--m navds-body-short"
          type="button"
        >
          knapp.neste
        </button>
      </div>
      <div
        aria-live="polite"
      />
    </fieldset>
    <fieldset
      aria-invalid="false"
      class="skjemagruppe navigasjon-rad"
    >
      <button
        class="navds-button navds-button--secondary navds-button--m navds-body-short"
        type="button"
      >
        knapp.avbryt
      </button>
      <div
        aria-live="polite"
      />
    </fieldset>
  </form>
</div>
`;
